tests/ltp/patches/fix_mkdir_mkdir02.patch

```diff
@@ -0,0 +1,74 @@
+	This change is necessary to run this test case in sgx-lkl
+	environment. Test case is modified to work in single
+	process environment by Removing exit code from child and 
+	wait code from parent. FORK_OR_VFORK return always 0 in
+	sgxlkl environment. Application in sgx-lkl environment
+	start with root user if this is the case Effective
+	capabilities of an user cleared when changing user
+	from root to non-root.Hence, test case is updated
+	to handle this scenario as well.
+
+diff --git a/testcases/kernel/syscalls/mkdir/mkdir02.c b/testcases/kernel/syscalls/mkdir/mkdir02.c
+index f09e3c33d..99f4324d5 100644
+--- a/testcases/kernel/syscalls/mkdir/mkdir02.c
++++ b/testcases/kernel/syscalls/mkdir/mkdir02.c
+@@ -17,6 +17,8 @@
+ #include <unistd.h>
+ #include <stdlib.h>
+ #include "tst_test.h"
++#include <sys/prctl.h>
++#include <linux/securebits.h>
+ 
+ #define TESTDIR1	"testdir1"
+ #define TESTDIR2	"testdir1/testdir2"
+@@ -54,10 +56,8 @@ static void verify_mkdir(void)
+ 		if (!fail)
+ 			tst_res(TPASS, "New dir inherited GID and S_ISGID");
+ 
+-		exit(0);
+ 	}
+ 
+-	tst_reap_children();
+ 	SAFE_RMDIR(TESTDIR2);
+ }
+ 
+@@ -67,6 +67,19 @@ static void setup(void)
+ 	struct passwd *pw;
+ 	struct stat buf;
+ 	pid_t pid;
++	int usr_secbits;
++
++	/*
++	 * if program started as a root user, avoid clearing of
```

Comment:
Stars looks ugly in the comment. Please fix. Better you can use multiple line // comments

Reply:
Ok, I will use the single line comments. modified the code and update patch

Reply:
Done